docs/02-app/02-api-reference/02-file-conventions/route-segment-config.mdx

```diff
@@ -109,7 +109,7 @@ export const revalidate = false
 > **Good to know**:
 >
 > - The revalidate value needs to be statically analyzable. For example `revalidate = 600` is valid, but `revalidate = 60 * 10` is not.
-> - The revalidate value is not available when using `runtime = 'edge'`.
+> - The revalidate value is not going to effect revalidation of the route for when using `runtime = 'edge'`, but can still effect the data cache.
```

Comment:
This should likely be an error then, right? If these two route segment configs are used together.

Reply:
This won't error, route segement `revalidate` can also effect the `revalidate` option in `fetch`, it's just not effecting the page since edge runtime has no ISR.

Reply:
Right, that is my point – you cannot use ISR with edge runtime, so rather than adding caveats here, we should consider just making it error on build if you do these together. Otherwise, you might expect it to work, and then have to check the docs to find this caveat.